packages/preview/songting-book/0.0.3/README.md

```diff
@@ -0,0 +1,126 @@
+# 松亭书籍排版模板 (SongTing Book Template)
+
+松亭书籍排版模板是一个为中文书籍排版设计的 Typst 模板，支持多种纸型，提供了完整的排版功能
```

Comment:
I think I skipped it in the previous reviews, but it would be nice if a brief description in English could be included here, just so that the non-Chinese part of the community can still understand what this is about. Having the rest of your README in Chinese is fine.